src/modules/crud.js

```diff
@@ -146,10 +153,46 @@ export const showEditInput = (paregraphElement) => {
   input.focus();
 };
 
+// eslint-disable-next-line no-unused-vars
+const removeTodo = (todoId, clear = false) => {
+  let currentList = getLocal();
+  if (!clear) {
+    currentList = currentList.filter((todo) => todo.id !== todoId);
+  } else {
+    currentList = currentList.filter((todo) => todo.id !== true);
+  }
+  updateLocal(currentList);
+};
+
+list.addEventListener('click', (e) => {
+  if (e.target.tagName === 'INPUT') {
+    const checkbox = e.target.checked;
+    const currentList = getLocal();
+    const task = currentList.find((todo) => todo.index === Number(e.target.id));
+    task.completed = checkbox;
+    updateLocal(currentList);
+  }
+});
+
+const clearCompleted = () => {
+  let currentList = getLocal();
+
+  currentList = currentList.filter((todo) => !todo.completed);
+  currentList = currentList.map((todo, index) => {
+    todo.index = index + 1;
+    return todo;
+  });
+  updateLocal(currentList);
+  loadTodo();
+};
+
+completed.addEventListener('click', clearCompleted);
+
 export const toggleComplete = (inputElement) => {
   if (inputElement.checked === false) {
     inputElement.parentElement.classList.remove('complete');
   } else {
     inputElement.parentElement.classList.add('complete');
+    inputElement.checked = true;
   }
-};
+};
```

Comment:
- [ ] **Required:** You have implemented the requirement for this milestone beautifully and yet maintain an impressive design 👏👏. However, you have broken some previous functionalities, the edit function of items has been broken.
https://github.com/zairdon20/Todo/blob/e823dfc41906ceb25eb93e5672b6bd7d978c22a0/src/modules/crud.js#L103-L144

![editing not saving to local storage](https://user-images.githubusercontent.com/27709832/150523941-854fb449-624c-406f-929e-56148c93c97c.gif)

Please fix this bug and request another review.